src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1039,6 +1041,10 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Generate and assign an Azure Active Directory Identity for this server'
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('minimal_tls_version',
+                   arg_type=get_enum_type(SqlServerMinimalTlsVersionType),
```

Comment:
You can abstract minimal_tls_version using CLIArgumentType to reuse the type for all `minimal_tls_version` parameters.
e.g. https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_params.py#L41